client/state/data-layer/http/index.js

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
I think it'd be useful to have simple tests for this like we do for `wpcom-http` [here](https://github.com/Automattic/wp-calypso/blob/eb1e8fb297d44f8a9bcef2e30da4da96415e2c0c/client/state/data-layer/wpcom-http/test/index.js), and at least a simple README.